src/main/java/frc/robot/subsystems/Acquisition.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.AcquisitionConstants;
+
+public class Acquisition extends SubsystemBase {
+    private static Acquisition instance;
+
+    private static CANSparkMax rearMotor = new CANSparkMax(AcquisitionConstants.frontMotorID, MotorType.kBrushless);
+    private static CANSparkMax frontMotor = new CANSparkMax(AcquisitionConstants.frontMotorID, MotorType.kBrushless);
+
+    private Acquisition() {
+        rearMotor.restoreFactoryDefaults();
+        frontMotor.restoreFactoryDefaults();
+        rearMotor.setIdleMode(IdleMode.kCoast);
+        frontMotor.setIdleMode(IdleMode.kCoast);
```

Comment:
```suggestion
        frontMotor.setIdleMode(IdleMode.kCoast);
        rearMotor.follow(frontMotor)
```
For now lets have these follow each other